### PR TITLE
Fix issue #40 - Add test and stories for media card

### PR DESCRIPTION
**Pull request by luminaire-dev**
## Related Issue
Resolves https://github.com/AnalogStudiosRI/www.blissri.com/issues/40

coming out of https://github.com/AnalogStudiosRI/www.blissri.com/issues/10 / https://github.com/AnalogStudiosRI/www.blissri.com/pull/21,

## Summary of Changes
1. [x] Add test cases for media card component
1. [x] Add stories for media card for both image and video

### CODE REVIEW

**Review by thescientist13 — APPROVED** [review]
Thank you so much for this!  Looks like there are just a few linting issues from running the build but otherwise this looks good to go from me once those are cleared up!

File: src/components/media-card/media-card.spec.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      document.body.appendChild(card);
+      await card.updateComplete;
+    });
+
+    it('should not be null', () => {
+      expect(card).not.equal(undefined);
+      expect(card.querySelectorAll('video').length).equal(1);
+    });
+
+    it('should render a video with a source', () => {
```
Comment (thescientist13) [inline src/components/media-card/media-card.spec.js:69]: For these specs, might be good to confirm that the `publicId` ends up in the `source` and `src` attributes for `<video>` and `` respectively, just to make sure the values gets passed through correctly.  I think just a simple `indexOf` check should suffice.

Comment (luminaire-dev) [reply src/components/media-card/media-card.spec.js]: Updated! :shipit:

File: src/components/media-card/media-card.stories.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+export default {
+  title: 'Components/Media Card'
+};
+
+const Template = ({ props }) => {
+  const { assest } = props;
+  return `
+    <bf-media-card
+     assest="${assest}"
```
Comment (thescientist13) [inline src/components/media-card/media-card.stories.js]: Looks like there was a small typo here on `assest` but with that little tweak and using some real `publicId`s this now looks good!
Screenshot 2024-05-25 at 2 55 00 PM
Screenshot 2024-05-25 at 2 55 04 PM

Comment (luminaire-dev) [reply src/components/media-card/media-card.stories.js]: Ohh good catch!

**Pull request merged**